pkg/apply/desiredset_process.go

```diff
@@ -466,22 +466,18 @@ func multiNamespaceList(ctx context.Context, namespaces []string, baseClient dyn
 	return wg.Wait()
 }
 
-func getDistinctNamespaces(objs map[objectset.ObjectKey]runtime.Object) (namespaces []string) {
+func getDistinctNamespaces(objs map[objectset.ObjectKey]runtime.Object) []string {
```

Comment:
Suggesting to apply the same change to `ObjectSet.Namespaces` - it's a bit of copy-paste there so may even extract helper func if you're up to it.

Reply:
I ended up tweaking things a bit more, wdyt?

Reply:
@brandond , I've noticed. ;) Like it. Just thought it would be great to add unit tests for `stringset.Set`.
Would like to have +1 approval anyway, will add couple more reviewers.

Reply:
Oh, didn't notice additional pushes (they arrived a few minutes after the comment). Will review later.

Reply:
Yeah, tests for the new stringset are a good idea. Will add later.